Author: Goedson Teixeira Paixao <goedson@debian.org>
Description: Fixes the gpaint.desktop file
 Change executable and icon filenames according to changes applied in the
 Debian packaging.

Index: b/gpaint.desktop
===================================================================
--- a/gpaint.desktop	2009-12-19 17:11:48.000000000 -0200
+++ b/gpaint.desktop	2009-12-19 17:12:10.000000000 -0200
@@ -3,11 +3,9 @@
 Comment=A small-scale painting program for GNOME, the GNU Desktop
 Comment[fr]=Un petit outil de dessin pour GNOME, le desktop GNU
 Comment[pt_BR]=Um pequeno programa de desenho para o GNOME, o Desktop GNU
-TryExec=gpaint-2
-Exec=gpaint-2
-Icon=gnome-logo-icon-transparent.png
+TryExec=gpaint
+Exec=gpaint
+Icon=gpaint.svg
 Terminal=0
 Type=Application
-
-
-
+Categories=Graphics;2DGraphics;RasterGraphics;
